lib/app/ui/grid/components/summary.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:f1_application/util/build_blur.dart';
+import 'package:flutter/material.dart';
+
+class Summary extends StatelessWidget {
+  final Widget childWidget;
+
+  const Summary(this.childWidget, {Key? key}) : super(key: key);
+

```

Comment:
Nekem furcsa itt, hogy a `childWidget` a konstruktor paraméter. Itt szerintem ez nem jó, mert valójában a Summary-nek így bármilyen widgetet is beadhatnánk, pl. egy textfieldet, labelt, tulajdonképp bármit. Viszont ezt nem így akarjuk használni (most még legalábbis), hanem úgy, hogy egy `Driver` listából képzett összesítőt jelenítünk meg. Más szóval, szerintem az nem jó minta, ha ezt a komponenst használó hívó fél dönti el, hogy a Summary hogyan jelenjen meg, mert sérülnek a felelősségek

Ennek fényében itt a jó interface az lenne, ha 

bejönne a driver lista és az a szolgáltatás (most a service), ami át tudja azt dolgozni a kívánt kimenetté (ebben az esetben a `grid_page.dart` -ban lévő `Widget _nations()` függvény is átköltözhet ide)

VAGY

ha már a végleges objektum jönne be (pl. a <String, Int> map amit most a service előállít), amit a komponens úgy jelenít meg, ahogy ő szeretne